packages/pg-v5/commands/credentials/destroy.js

```diff
@@ -15,8 +15,9 @@ async function run(context, heroku) {
   }
```

Comment:
It may be good to change the messaging for numbered essentials around destroying the default credential. Right now it says default credential cant be destroyed, but perhaps it should be "You can’t perform this operation on Essential-tier databases."?

Reply:
Has any of this language been checked with CX?